### PR TITLE
Add env vars to control log output.

### PR DESCRIPTION
**Pull request by inelpandzic**
Currently, the PSMDB operator had configured only structured logs which, when looking in the console, is really hard to parse.

Just as we have it in PSO, I added two env vars to control the log output, log level, and log formatting (structured or console).

```
          env:
            - name: LOG_STRUCTURED
              value: 'false'
            - name: LOG_LEVEL
              value: INFO
```

### CODE REVIEW

**Review by hors — CHANGES_REQUESTED** [review]
@inelpandzic please create the task for these changes and at it to commit message

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| We run 0 out of 30|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/995/commits/9787db5cc2a53a56b54c3966dd9ffe65213119dd
image: `perconalab/percona-server-mongodb-operator:PR-995-9787db5c`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| one-pod | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/one-pod.log) |
| storage | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/storage.log) |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/init-deploy.log) |
| cross-site-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/cross-site-sharded.log) |
| upgrade-consistency | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/upgrade-consistency.log) |
| monitoring-2-0 | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/monitoring-2-0.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/demand-backup.log) |
| limits | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/limits.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/self-healing.log) |
| self-healing-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/self-healing-chaos.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/operator-self-healing.log) |
| operator-self-healing-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/operator-self-healing-chaos.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/scheduled-backup.log) |
| scaling | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/scaling.log) |
| arbiter | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/arbiter.log) |
| demand-backup-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/demand-backup-sharded.log) |
| security-context | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/security-context.log) |
| service-per-pod | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/service-per-pod.log) |
| upgrade | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/upgrade.log) |
| rs-shard-migration | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/rs-shard-migration.log) |
| liveness | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/liveness.log) |
| upgrade-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/upgrade-sharded.log) |
| smart-update | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/smart-update.log) |
| version-service | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/version-service.log) |
| users | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/users.log) |
| pitr | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/pitr.log) |
| data-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/data-sharded.log) |
| pitr-sharded | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/pitr-sharded.log) |
| non-voting | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/non-voting.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-995/9dd81fbf/demand-backup-eks-credentials.log) |
| We run 30 out of 30|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/995/commits/9dd81fbfbf899afca5b8e03a64a896edb90d1b0a
image: `perconalab/percona-server-mongodb-operator:PR-995-9dd81fbf`
